relations.go

```diff
@@ -42,6 +42,19 @@ func (relations *Relations) Add(parent string, child string) {
 	relations.ChildToParents[child][parent] = true
 }
 
+// Remove removes relation
+func (relations *Relations) Remove(parent string, child string) {
+	if _, ok := relations.ParentToChildren[parent]; !ok {
```

Comment:
What about rather:

```go
if _, ok := relations.ParentToChildren[parent]; ok {
    delete(relations.ParentToChildren[parent], child)
}
```

And the same for the next if?
```

Reply:
Good one